portal-ui/src/screens/Console/Tenants/utils.ts

```diff
@@ -35,3 +35,7 @@ export const getRandomString = function (length = 16): string {
   }
   return retval;
 };
+
+export const imagePattern = "^[a-zA-Z0-9-_./:]{1,253}$";
+
+export const resourcePattern = "[0-9]*";
```

Comment:
This should be calles numeric pattern  as we are using it in other parts of the application too
```suggestion
export const numericPattern = "[0-9]*";
```